src/components/appointments/CreateAppointment/Summary.spec.js

```diff
@@ -18,22 +18,37 @@ import {
   mockAppointmentPost,
   mockAppointmentProductsGet,
 } from '../mocks';
+import {buildContextValue} from './CreateAppointmentState';
 import Summary from './Summary';
 
 const renderSummary = errorHandler => {
   const appointmentData = {
-    products: [{productId: '166a5c79', amount: 1}],
-    location: '1396f17c',
-    date: '2023-07-12',
-    datetime: `2023-07-12T08:00:00Z`,
-    lastName: 'Kundera',
-    dateOfBirth: '1929-04-01',
-    email: 'milan@kundera.cz',
-    phone: '12345678',
-    bsn: '123456782',
-    gender: 'M',
+    producten: {
+      products: [{productId: '166a5c79', amount: 1}],
+    },
+    kalender: {
+      location: '1396f17c',
+      date: '2023-07-12',
+      datetime: `2023-07-12T08:00:00Z`,
+    },
+    contactgegevens: {
+      lastName: 'Kundera',
+      dateOfBirth: '1929-04-01',
+      email: 'milan@kundera.cz',
+      phone: '12345678',
+      bsn: '123456782',
```

Comment:
I'd expect an Eidas psuedo ID here. :stuck_out_tongue_winking_eye:  Sorry for being so unbearable.